scm/driver/github/webhook.go

```diff
@@ -63,7 +63,7 @@ func (s *webhookService) Parse(req *http.Request, fn scm.SecretFunc) (scm.Webhoo
 		return hook, nil
 	}
 
-	sig := req.Header.Get("X-Hub-Signature")
+	sig := req.Header.Get("X-Hub-Signature-256")
```

Comment:
We have some folks using this library with older versions of GitHub Enterprise.  In the interest of backward compatibility and ensuring we don't cause any regressions (for our Drone users) I am thinking we should fallback to the original header.

```Go
sig := req.Header.Get("X-Hub-Signature-256")
if sig == "" {
  sig = req.Header.Get("X-Hub-Signature")
}
```